react-effects/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom/client';
+import readItems from './read.js';
+
+function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+
+  // Your code here:
+  //  - Read the items using `readItems` and update state so the list displays
+
+  const storedPromise = readItems();
```

Comment:
You want to call this _inside_ `useEffect`. Otherwise it will run on every render and will make numerous backend calls.